flink-formats/flink-avro/pom.xml

```diff
@@ -106,6 +106,15 @@ under the License.
 			<scope>test</scope>
 			<type>test-jar</type>
 		</dependency>
+
+		<!-- We need this for the patched FlinkScalaKryoInstantiator -->
```

Comment:
Why do we need the `FlinkScalaKryoInstantiator`. The serializers, including Kryo, should work without it. This indicates some problem in the scope of the flink-avro dependenies...